src/shapes.ts

```diff
@@ -539,16 +544,18 @@ export class Label extends Shape {
   private drawText(context: CanvasRenderingContext2D): void {
     const contextMatrix = context.getTransform();
     context.resetTransform();
-    this.text.draw(context);
+    this.text.fontsize = null;
+    this.text.rowIndices = [];
+    try { 
+      this.text.draw(context);
+    } catch (TypeError) {}
```

Comment:
It seems weird to do nothing when catching an error. What is the usecase, here ?

Reply:
It is to avoid a bug when objects are too small and their dimensions cannot be handled anymore by canvas.
Catching this error allows to play from very wide pics to very small not drawn pics without crashing web browser

Reply:
Alright, I think stating it as a comment in the catch statement could help next devs to understand it